main.js

```diff
@@ -16,3 +17,137 @@ listItems.forEach((element) => {
     cover.classList.remove('d-flex');
   });
 });
+
+const projectCards = [
+  {
+    title: 'Keeping track of hundreds of components',
+    image: './IMG/modal-popup.png',
+    technologies: ['Ruby on rails', 'CSS', 'JavaScript', 'Bootstrap', 'CodePen', 'Terminal', 'CodeKit', 'GitHub'],
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dumm. Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dumm',
+    button: 'See Project',
+    links: ['https://jeffkay16.github.io/Portfolio-Website/', 'https://github.com/JEFFKAY16/Portfolio-Website.git'],
+  },
+];
+const [project] = projectCards;
+const popupSection = document.querySelector('.popup-section');
+popupSection.innerHTML = ` 
+<div class="popup-card">
+    <i class="fa fa-times popup-close-btn" aria-hidden="true"></i>
```

Comment:
- [ ] I do see the Icon is here ut it's not displaying.

Reply:
Kindly make reference to the above comment.

In Addition, you might realize that users do not adjust the screens as they're fixed by design. i.e, when a user opens the platform on a mobile or desktop (The only two designs we're given, excluding the mentioned tablet mode [here](https://github.com/JEFFKAY16/Portfolio-Website/pull/15#discussion_r724900025), they will only be utilizing a popup respectively designed to their device ratio.

You must close the popup and resize your device to mobile, and try reopening it back in order to get the mobile popup preview. 

Kindly comment and revert post perusal.

Reply:
@dasileker 

Thank you once more. I truthfully think you're not being professional- you're blocking me from going forward. I really appreciate your feedback and I'm seeing this as an opportunity to learn and improve. You need to build on the previous reviewer's suggested changes, which I already did. I have already attached the screenshots which the previous reviewer has acknowledged. You have also acknowledged that the internet issue which failed to load some fonts was on your side. The media queries you're now raising are different from what you requested. Firstly, I'm only supposed to work on the Desktop and Mobile only. Imagine if I had made the changes, requested another review then you would have again raised another issue which in my opinion is unfair and very unprofessional. Kindly approve my work, please.